doc/api/next_api_changes/development/00001-EFS.rst

```diff
@@ -0,0 +1,6 @@
+- The signature and implementation of the `set_hover()` and `get_hover()` methods of the `Artist` class, as well as the `HoverEvent(Event)` Class, have been added:
```

Comment:
```suggestion
- The signature and implementation of the ``set_hover()`` and ``get_hover()`` methods of the ``Artist`` class, as well as the ``HoverEvent(Event)`` Class, have been added:
```

Throughout this file I think you mostly want double backticks (format as code) instead of single backticks (try to find a matching link). 

This file also needs a title (with `~` underline) and should be named `25831-EFS.rst` (reference to the PR number)